spec/lib/topics_filter_spec.rb

```diff
@@ -617,6 +617,18 @@
       fab!(:closed_topic) { Fabricate(:topic, closed: true) }
       fab!(:archived_topic) { Fabricate(:topic, archived: true) }
       fab!(:deleted_topic_id) { Fabricate(:topic, deleted_at: Time.zone.now).id }
+      fab!(:foobar_topic) { Fabricate(:topic, closed: true, word_count: 42) }
+
+      it "supports custom status filters" do
+        TopicsFilter.add_filter_by_status("foobar") { |scope| scope.where("word_count = 42") }
```

Comment:
We need to clean up this state so that it doesn't leak across tests.

Reply:
Good catch. Will fix 👍

Reply:
Will be fixed in https://github.com/discourse/discourse/pull/26803